pages/doc.js

```diff
@@ -20,6 +20,8 @@ import styled from 'styled-components'
 import { media } from '../src/styles'
 // sidebar data and helpers
 import sidebar, { getItemByPath } from '../src/Documentation/SidebarMenu/helper'
+// constants
+import { HEADER } from '../consts'
 
 const ROOT_ELEMENT = 'bodybag'
 const SIDEBAR_MENU = 'sidebar-menu'
```

Comment:
Maybe we should move this id's to the consts file too?

Reply:
Yeah, I have plans to. But not in this PR.